assets/src/scripts/monitoring-location/components/hydrograph/graph-brush.js

```diff
@@ -41,11 +54,19 @@ export const drawGraphBrush = function(container, store) {
         .call(link(store,(elem, layout) => {
                 elem.attr('viewBox', `0 0 ${layout.width + layout.margin.left + layout.margin.right} ${layout.height + layout.margin.bottom + layout.margin.top}`);
             }, getBrushLayout
-            ))
+        ))
         .call(svg => {
+            svg.append('text')
+                .classed('brush-text-hint', true)
+                .text('drag handles to change timeframe')
+                .attr('text-anchor', 'middle')
+                .attr('font-size', 'smaller')
```

Comment:
I would like the font-size to be specified in the style sheet, preferably using a USWDS design token.

Reply:
Will do.